src/config/validation/taskSchema.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from "zod";
+
+export const taskSchema = z.object({
+    id: z.string().optional(),
+    title: z.string().min(5, { message: "Task title is required" }).max(30, { message: "Task title is too long" }),
+    priority: z.enum(["Urgent", "High", "Normal", "Low"]),
+    storyPoints: z.coerce.number().min(1, { message: "Number must be a number between 1 and 10" }).max(20, { message: "Number must be a number between 1 and 20" }),
```

Comment:
Why coerce is needed?

Reply:
The problem is that input type="number" when is accessed by the form, for some reason is taking it as string, so this parses to string.